prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -313,8 +312,8 @@ ssize_t rxr_pkt_post_ctrl_once(struct rxr_ep *rxr_ep, int entry_type, void *x_en
 	if (!pkt_entry)
 		return -FI_EAGAIN;
 
-	send.iov_count = 0;
-	pkt_entry->send = &send;
+	pkt_entry->send = malloc(sizeof(struct rxr_pkt_sendv));
```

Comment:
I worry that malloc and free for each packet would impact performance.

maybe use another bufpool? we can also move some variables used only locally to that pool

Reply:
Had a new bufpool for rxr_pkt_sendv